### ISSUE TITLE
Failure in tests

### ISSUE THREAD

**Issue by guimard** [component/merge-descriptors#8]
Using Node >=10, tests fail. Here is the patch:
```diff
--- a/test/merge.js
+++ b/test/merge.js
@@ -16,21 +16,21 @@
   describe('arguments', function () {
     describe('dest', function () {
       it('should be required', function () {
-        assert.throws(merge.bind(null, undefined), 'argument dest is required')
+        assert.throws(merge.bind(null, undefined), 'TypeError: argument dest is required')
       })

       it('should reject null', function () {
-        assert.throws(merge.bind(null, null), 'argument dest is required')
+        assert.throws(merge.bind(null, null), 'TypeError: argument dest is required')
       })
     })

     describe('src', function () {
       it('should be required', function () {
-        assert.throws(merge.bind(null, {}, undefined), 'argument src is required')
+        assert.throws(merge.bind(null, {}, undefined), 'TypeError: argument src is required')
       })

       it('should reject null', function () {
-        assert.throws(merge.bind(null, {}, null), 'argument dest is required')
+        assert.throws(merge.bind(null, {}, null), 'TypeError: argument dest is required')
       })
     })
```

**Comment by dougwilson** [comment]
Hi! Thank you. What specific version of Node.js is that? The tests are passing in Node.js in Travis CI and that patch does not even apply on master.

Here was the last Node.js 10 run in Travis: https://travis-ci.org/component/merge-descriptors/jobs/478397844

**Comment by guimard** [comment]
Hello, I'm using Node 10.15

**Comment by dougwilson** [comment]
Thanks. Node.js 10.15 is the version used in that Travis CI run.

So it would seem that the current tests do pass on Node.js 10. I think that since the patch you provided doesn't even apply (I cannot apply it to master, the patch program won't let me), maybe you're using an older version of the test suite?

**Comment by dougwilson** [comment]
I think the question may have come through poorly. I'm not sure what you're asking there. Can you rephrase / elaborate?

**Comment by guimard** [comment]
Sorry (I'm not fluently in English). Error string seems different here (Debian/unstable) than in your configuration. I thought that TypeError.js might be different here, but I don't know from which module it comes.

**Comment by dougwilson** [comment]
Gotcha. I'm not sure. Perhaps if I could reproduce the error I could try and debug it and see. Do you know how I can reproduce the issue you're having?

**Comment by guimard** [comment]
Do you have a Ubuntu or Debian environment ?

**Comment by guimard** [comment]
If yes, in an empty directory, launch this (as root) :
```shell
# debootstrap unstable ./ http://deb.debian.org/debian
# tar xzf /path/to/your/module
# chroot .
# apt-get install nodejs npm debhelper mocha
# cd merge-descriptors*
# mocha --reporter spec --bail --check-leaks test/
```

**Comment by dougwilson** [comment]
Not off hand, but maybe I can get something setup. I have Windows 10 machines if that helps. I will look into setting it up in the coming weeks.

Perhaps if there also is a patch I can apply, I can apply that patch here, but the given patch above does not apply to the code i  this repo.

**Comment by guimard** [comment]
@dougwilson : the more simple is probably to replace assert.throws by a "assert.throwsMatch" if exists

**Comment by dougwilson** [comment]
There is no such thing as throwsMatch in Node.js 10.15: https://nodejs.org/dist/latest-v10.x/docs/api/assert.html
